classes/app.rb

```diff
@@ -0,0 +1,120 @@
+require_relative './student'
+require_relative './teacher'
+require_relative './book'
+require_relative './rental'
+require_relative './main'
+
+def list_all_books
+  Book.all_instances.each do |book|
+    puts "Title: \"#{book.title}\", Author: #{book.author}"
+  end
+  puts ''
+end
+
+def list_all_people
+  Person.all_instances.each do |person|
+    puts "[#{person.class.name}] Name: #{person.name}, ID:#{person.id}, Age:#{person.age}"
+  end
+  puts ''
+end
+
+def create_a_person
+  puts 'Do you want to create a student (1) or a teacher (2)'
+  person_choice = gets.chomp.to_i
+
+  case person_choice
+  when 1
+    create_student
+
+  when 2
+    create_teacher
+
+  else
+    puts 'Please enter an appropriate number'
+    create_a_person
+  end
+
+  main_menu
+end
+
+def create_student
+  puts 'Please enter the following details for the student'
+  print 'Name: '
+  student_name = gets.chomp
+  print 'Age: '
+  student_age = gets.chomp
+  print 'Classroom: '
+  student_classroom = gets.chomp
```

Comment:
- [ ] Kindly consider removing this as asking input for classroom is not required in the example video.